packages/components/src/Toggle/index.js

```diff
@@ -1,5 +1,7 @@
 import Toggle from './Toggle.component';
+import LabelToggle from './LabelToggle/LabelToggle.component';
 
 export default Toggle;
 
 export { Checkbox } from './Checkbox';
+export { LabelToggle };
```

Comment:
WDYT of `export Toggle.Multiple` instead?

Reply:
good for me